syft/workers/abstract.py

```diff
@@ -36,3 +38,31 @@ def _recv_msg(self, message: bin):
             message: The binary message being received.
         """
         pass
+
```

Comment:
Must be defined by worker classes, but this leaves the communication and messaging methods completely unspecified, so that various worker types can fill them in.